custom_components/tuya_local/devices/himox_h06_purifier.yaml

```diff
@@ -0,0 +1,97 @@
+name: himox h06
+products:
+ - id: 80256465f4cfa2c5ec84
+primary_entity:
+  entity: fan
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+    - id: 4
+      type: string
+      name: speed
+      mapping:
+        - dps_val: "low"
+          value: 33.34
+        - dps_val: "mid"
+          value: 66.67
+        - dps_val: "high"
+          value: 100
+secondary_entities:
+  - entity: sensor
+    name: active_filter_life
```

Comment:
Entity names should be as they should appear in the UI.  The names will be automatically downcased and slugified for use in the default entity ID.  So for this, Active filter life (is it an active filter?)